dao/accumulo.rya/src/main/java/org/apache/rya/accumulo/AbstractAccumuloRdfConfigurationBuilder.java

```diff
@@ -0,0 +1,209 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.rya.accumulo;
+
+import org.apache.rya.api.RdfCloudTripleStoreConfiguration;
+import org.apache.rya.api.RdfCloudTripleStoreConfigurationBuilder;
+
+import com.google.common.base.Preconditions;
+
+/**
+ * This builder class will set all of the core Accumulo-backed Rya configuration
+ * parameters. Any builder extending this class will have setter methods for all
+ * of the necessary parameters to connect to an Accumulo backed Rya instance.
+ *
+ * @param <B>
+ *            - builder returned by setter methods extending this class
+ * @param <C>
+ *            - configuration object constructed by the builder extending this
+ *            class
+ */
+public abstract class AbstractAccumuloRdfConfigurationBuilder<B extends AbstractAccumuloRdfConfigurationBuilder<B, C>, C extends AccumuloRdfConfiguration>
```

Comment:
This class is extended by the AccumuloIndexingConfigBuilder and new functionality is added there.